perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -440,6 +454,7 @@ def Prepare(self, vm):
 
     # Get a list of zones and pick one that's different from the zone VM is in.
     new_subnet_zone = None
+    storage_size = self._ValidateSize(FLAGS.storage_size)
```

Comment:
It doesn't seem necessary to return anything from the validatesize() function, if it never changes the size.